src/server.ts

```diff
@@ -19,6 +19,13 @@ import { SorobanRpc } from "./soroban_rpc";
 
 export const SUBMIT_TRANSACTION_TIMEOUT = 60 * 1000;
 
+export interface GetEventsRequest {
+  startLedger?: number;
+  filters?: SorobanRpc.EventFilter[];
```

Comment:
startLedger and filters are still both required here, afaik (or we should update the docs to mark them optional)

Reply:
Is that reflected in soroban-rpc as well?